packages/website/src/components/Playground.tsx

```diff
@@ -8,10 +8,14 @@ import Loader from './layout/Loader';
 
 import useHashState from './hooks/useHashState';
 import OptionsSelector from './OptionsSelector';
-import ASTViewer from './ast/ASTViewer';
 import { LoadingEditor } from './editor/LoadingEditor';
 import { EditorEmbed } from './editor/EditorEmbed';
-import type { RuleDetails } from './types';
+import { shallowEqual } from './lib/shallowEqual';
+
+import ASTEsViewer from './ASTEsViewer';
+import ASTTsViewer from './ASTTsViewer';
+
+import type { RuleDetails, SelectedRange } from './types';
```

Comment:
random thought completely unrelated to this PR that we can address separately if we want.
We have `eslint-plugin-simple-import-sort` [installed and enabled on the ast-spec codebase](https://github.com/typescript-eslint/typescript-eslint/blob/f4b64c5bf17cd01b63e3806105d0bf1091ffae70/.eslintrc.js#L298).
We should probably standardise the website package with the same import sorting